volcano.tf

```diff
@@ -0,0 +1,66 @@
+################################################################################
+# Volcano Scheduler for TorchX
```

Comment:
Replace this text with "Volcano Batch Scheduler" 
Its not just used for TorchX as its used with many other frameworks

Reply:
Done